src/styles/index.scss

```diff
@@ -1,3 +1,64 @@
 body {
+  width: 100%;
+  height: 100%;
   margin: 0;
+  padding: 0;
+}
+
+.stopwatch {
+  position: absolute;
+  inset: 0;
+  margin: auto;
+  width: 80vmin;
+  height: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minute-hand {
+    margin: auto;
+    position: absolute;
+    left: 38.5vmin;
+    top: 20vmin;
+    width: 3vmin;
+    height: 20vmin;
+    background-color: #0700ff;
+    z-index: -1;
+
+    @include animation-minute-hand;
+
+    animation-duration: 3600s;
+
+    &--speed-up {
+      animation-duration: 600s;
+    }
+  }
+
+  &__second-hand {
+    margin: auto;
+    position: absolute;
+    left: 39.25vmin;
+    width: 1.5vmin;
+    height: 38vmin;
+    background-color: #2c8000;
+    z-index: 0;
+
+    @include animation-second-hand;
+
+    animation-duration: 60s;
+
+    &--speed-up {
+      animation-duration: 10s;
+    }
+  }
+
+  &__axis {
+    margin: auto;
+    position: absolute;
+    inset: 0;
+    width: 5vmin;
+    height: 5vmin;
+    background-color: #f6a603;
+    border-radius: 50%;
+    z-index: 1;
+  }
```

Comment:
Same suggestion as above, a brief comment explaining the purpose of the `__axis` class would be helpful.